bungee/src/main/java/dev/endoy/helpers/bungee/BungeeEndoyApplication.java

```diff
@@ -53,6 +55,12 @@ public File getDataFolder()
     @Override
     public void registerListeners( Object listenersInstance )
     {
+        if ( Arrays.stream( listenersInstance.getClass().getDeclaredMethods() )
```

Comment:
ooohh nice idea to do this here

perhaps it's also a good idea to check if the class implements Listener here?

Reply:
Should be there now